Manager/MenuManager.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Grossum\MenuBundle\Manager;
+
+use Grossum\MenuBundle\Handler\MenuHandlerInterface;
+
+class MenuManager
+{
+    /**
+     * @var MenuHandlerInterface[]
+     */
+    protected $handlers = [];
+
+    /**
+     * @param MenuHandlerInterface $menuHandler
+     */
+    public function addMenuHandler(MenuHandlerInterface $menuHandler)
+    {
+        $this->handlers[$menuHandler->getIdentifierClass()] = $menuHandler;
+    }
+
+    /**
+     * @param string $className
+     * @return MenuHandlerInterface
+     */
+    public function getMenuHandler($className)
+    {
+        if (!array_key_exists((string)$className, $this->handlers)) {
```

Comment:
But in https://github.com/GrossumUA/MenuBundle/pull/2/files#diff-2eda2f5c6527c6f9d8cad045ebf0e8a7R29 you waiting string